### ISSUE TITLE
[Accessibility] The 'Edit' element in ComboBox with DropDownStyle(DropDown, Simple) should support ValuePattern and TextPattern

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#4409]
* .NET Core Version: 
.Net SDK 6.0.100-alpha.1.20623.10

* Have you experienced this same bug with .NET Framework?: 
No

**More info:**
This is a regression issue, cannot repro earlier 6.0 build, regression from the fix of pull: [4185](https://github.com/dotnet/winforms/pull/4185)

**Minimal repro:**
1. Create a Winforms .Net Core app with targeting 6.0
2. Add 2 comboBox controls with several items, and set their DropDownStyle property to DropDown, Simple separately
3. Check related Value pattern and Text pattern in Inspect for ComboBox Edit part, and in Narrator, check its behavior

**Actual behavior:**
In Inspect, LegacyIAccessible.Name IsTextPattemAvailable, IsTextPattem2Available, IsValuePattemAvailable properties are changed, and Missing Value.IsReadOnly, Value.Value, IsContentElement properties
![image](https://user-images.githubusercontent.com/26474449/103057869-aeef3a00-45db-11eb-9abb-50cdc8a5429b.png)
In Narrator, it cannot announce comboBox edit part
![image](https://user-images.githubusercontent.com/26474449/103057897-c0384680-45db-11eb-9cb6-eca451703fd7.png)

**Expected behavior:**
In Inspect, it supports ValuePattern and TextPattern for comboBox edit element
![image](https://user-images.githubusercontent.com/26474449/103058040-1c02cf80-45dc-11eb-8966-df1d29ac5cc3.png)
In Narrator, it can announce comboBox edit part.
![image](https://user-images.githubusercontent.com/26474449/103057990-f5449900-45db-11eb-97d2-19de2ad69b93.png)

**Comment by Olina-Zhang** [comment]
Verified this issue in the latest .Net 6.0 SDK build: 6.0.100-Preview.4.21180.17, it is fixed for comboBox with DropDownStyle: dropdown, simple, the 'Edit' element in ComboBox has supported ValuePattern and TextPattern.
![image](https://user-images.githubusercontent.com/26474449/113120001-ae51e980-9243-11eb-98ca-df3afd18f067.png)
